_infra/helm/collection-instrument/templates/deployment.yaml

```diff
@@ -195,5 +195,11 @@ spec:
             value: "{{ .Values.gcp.seftProject }}"
           - name: SEFT_PUBSUB_TOPIC
             value: "{{ .Values.gcp.seftTopic }}"
+          - name: SEFT_CI_BUCKET_NAME
+            value: "{{ .Values.gcpDev.bucket.name }}"
+          - name: GOOGLE_CLOUD_PROJECT
```

Comment:
The project is defined twice. We should only have 1 item and have both uses of it use the same environment variable.

Reply:
I think we need it though, as the pub sub topic for SDX is in a different project